tests/integration/goldens/redis/tests/unit/gapic/redis_v1/test_cloud_redis.py

```diff
@@ -8371,6 +8151,237 @@ def test_transport_kind_rest():
     assert transport.kind == "rest"
 
 
+def test_list_instances_rest_bad_request(request_type=cloud_redis.ListInstancesRequest):
```

Comment:
Do any of the goldens test the `await_prefix`, `async_prefix`, etc?

Reply:
Not here but in one of the follow up PRs where we turn on this test for async.